resources/lang/ind.ini

```diff
@@ -1,46 +1,112 @@
+command.info.title = "Informasi"
+command.limit.current = "Batas saat ini: {%0}"
+command.limit.set = "Batas perubahan blok disetel ke {%0}"
+command.setrange.current = "Kisaran saat ini: {%0}"
+command.setrange.set = "Rentang alat diatur ke {%0}"
+command.biomeinfo.attarget = "Bioma target"
+command.biomeinfo.atposition = "Bioma di posisi"
+command.biomeinfo.result = "{%0} bioma ditemukan dalam seleksi"
+command.biomeinfo.result.line = "ID: {%0} Nama: {%1}"
+command.biomelist.title = "Daftar Biome"
+command.biomelist.result.line = "ID: {%0} Nama: {%1}"
+command.brushname.set = "Nama kuas ditetapkan ke \"{%0}\""
+command.clearclipboard.cleared = "Clipboard dihapus"
+command.flip.try = "Mencoba membalik clipboard dengan {%0}"
+command.flip.success = "Papan klip terbalik sukses"
+command.rotate.try = "Mencoba memutar clipboard dengan {%0} derajat"
+command.rotate.success = "Papan klip berhasil diputar"
+command.history.cleared = "Riwayat dihapus"
+command.listchunks.found = "{%0} potongan ditemukan dalam pilihan"
+command.size = "Ukuran pilihan"
+; selection
+selection.pos1.set = "Posisi 1 diatur ke X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "Posisi 2 diatur ke X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "Tidak ada yang dibatalkan"
+session.undo.left = "Anda memiliki {%0} membatalkan tindakan yang tersisa"
+session.redo.none = "Tidak ada yang harus diulang"
+session.redo.left = "Anda memiliki {%0} ulangi tindakan yang tersisa"
+session.brush.added = "Ditambahkan {%0} ke sesi"
+session.brush.deleted = "Dihapus {%0} (UUID {%1})"
+session.brush.removed = "Dihapus {%0} (UUID {%1})"
+session.language.set = "Berhasil mengatur bahasa ke {%0}"
+session.language.notfound = "Bahasa {%0} tidak ditemukan, mengatur ulang ke default"
+; task
+task.copy.success = "Salin Async berhasil, ambil {%0}, salin {% 1} blok dari {%2}."
+task.count.success = "Analisis async berhasil, ambil {%0}"
+task.count.result = "{%0} blok ditemukan dalam total {%1} blok"
+task.fill.success = "Async Fill berhasil, ambil {%0}, {%1} blok dari {%2} diubah."
+task.replace.success = "Async Replace berhasil, mengambil {%0}, {%1} blok dari {%2} diubah."
+task.revert.undo.success = "Async Undo berhasil, ambil {%0}, {%1} blok dari {%2} diubah."
+task.revert.redo.success = "Async Redo berhasil, ambil {%0}, {%1} blok dari {%2} diubah."
+; flags
+flags.keepexistingblocks = "Simpan blok yang ada"
+flags.keepair = "Jaga udara"
+flags.hollow = "Hollow"
+flags.hollowclosed = "Berongga dengan ujung tertutup"
+flags.natural = "Alami"
+; tools
+; wand tool
+tool.wand = "Wand"
+tool.wand.lore.1 = "Klik kiri sebuah blok untuk mengatur posisi 1 dari suatu pilihan"
+tool.wand.lore.2 = "Klik kanan sebuah blok untuk mengatur posisi 2 dari suatu pilihan"
+tool.wand.lore.3 = "Gunakan // togglewand untuk mengganti fungsionalitasnya"
+tool.wand.disabled = "Alat tongkat dinonaktifkan. Gunakan // togglewand untuk mengaktifkannya kembali"
+tool.wand.setenabled = "Alat tongkat sekarang {% }!"
```

Comment:
Please make sure to keep the number - it is a placeholder for "enabled/disabled"